src/app/CommandHandler.cpp

```diff
@@ -540,6 +540,17 @@ CHIP_ERROR CommandHandler::FinishStatus()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CommandHandler::ResetResponse()
```

Comment:
This does not look right to me.  What if we have multiple command responses we are trying to encode?  This is going to lose the already-encoded ones if a later one fails to encode, no?

Reply:
We only support one command in a transactions for now, so this won't be an issue.

Changed ResetResponse to RollbackResponse to support multiple commands in the future.

Reply:
> We only support one command in a transactions for now

That is ... not true.  We absolutely support multiple commands.  See `CommandHandler::ProcessInvokeRequest`:
```
    while (CHIP_NO_ERROR == (err = invokeRequestsReader.Next()))
    {
        CommandDataIB::Parser commandData;
        ReturnErrorOnFailure(commandData.Init(invokeRequestsReader));
...
            ReturnErrorOnFailure(ProcessCommandDataIB(commandData));
```

Reply:
We should nix that bit of logic and align the entirety of the code base against the expectations for 1.0, which is '1 command in a given invoke interaction'.

Reply:
PR: https://github.com/project-chip/connectedhomeip/pull/16338

The PR above should not conflict with this PR, and I think these two PRs can be merged in parallel.

Reply:
#16338 is merged so this PR should be ready now.